cluster-autoscaler/cloudprovider/huaweicloud/README.md

```diff
@@ -1,19 +1,12 @@
 # Cluster Autoscaler on Huawei Cloud 
 
 ## Overview
-The cluster autoscaler for [Huawei ServiceStage](https://www.huaweicloud.com/intl/en-us/product/servicestage.html) scales worker nodes within any
-specified container cluster's node pool where the `Autoscaler` label is on. 
+The cluster autoscaler for [Huawei ServiceStage](https://www.huaweicloud.com/intl/en-us/product/servicestage.html) works with self-built Kubernetes cluster on [Huaweicloud ECS](https://www.huaweicloud.com/intl/en-us/product/ecs.html) and
```

Comment:
The `autoscaler` is not for `Huawei ServiceStage`, it can work with or used by it.